doc/installation.rst

```diff
@@ -249,7 +282,9 @@ To launch Spyder after installing, ensure your environment is activated and run
 Updating Spyder
 ===============
 
-If you installed Spyder through Anaconda (recommended), WinPython, MacPorts, or your system package manager, update using those same methods.
+To update a copy of Spyder from our :ref:`standalone installers<standalone_installers_ref>` on Windows and macOS, you'll currently need to download and install the latest installer (if on Windows, make sure to uninstall it first from Control Panel/System Settings).
```

Comment:
I know the "a copy" makes it technically right but it might be a bit confusing for users. Can we not just say "TO update Spyder"?

Reply:
I'm not 100% happy with it either; "a copy" helps clarify that its referring to a specific Spyder installation (they may, and quite likely will have multiple if they have installed Anaconda) and was particularly necessary with the previous syntax. However, I rewrote it as proposed below to avoid a lot of word duplication with "install" variants and simplify it a bit, so "a copy of" is less necessary (though I still personally feel it is a little clearer and more explicit with it).

```suggestion
To update Spyder installed via our :ref:`standalone packages<standalone_installers_ref>` on Windows and macOS, you'll currently need to manually download and install the latest release (if on Windows, make sure to remove the old version first from Control Panel/System Settings).
```

Reply:
Sounds better like this.

Reply:
Okay